### PR TITLE
Add missing test for `SMALLINT` and `TINYINT` in TestMathFunctions

### PR DESCRIPTION
**Pull request by fiedukow**

### CODE REVIEW

**Comment by fiedukow** [comment]
@haozhun Can you review this maybe or pass to someone who will?

File: presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java [hunk 1]
```diff
@@ -439,7 +439,7 @@ public static long roundSmallint(@SqlType(StandardTypes.SMALLINT) long num, @Sql
     @Description("round to nearest integer")
     @ScalarFunction("round")
     @SqlType(StandardTypes.INTEGER)
-    public static long roundInteger(@SqlType(StandardTypes.INTEGER) long num, @SqlType(StandardTypes.INTEGER) long decimals)
+    public static long roundInteger(@SqlType(StandardTypes.INTEGER) long num, @SqlType(StandardTypes.BIGINT) long decimals)
```
Comment (raghavsethi) [inline presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java]: This was actually by design. If you happen to want to round (int, int):int, the function will be resolved to round(int, bigint):int. However, with your design, if you wanted to round(int, bigint):int, the only matching function would be round(bigint, bigint):bigint, which gives you an unexpected result type.

Comment (fiedukow) [inline presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java]: I think that is incorrect. I will check that and add tests covering that case tomorrow morning (CEST).

Comment (raghavsethi) [inline presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java]: Oops sorry - I read it wrong. The original was broken, yours has the behavior I intended.

Comment (fiedukow) [inline presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java]: Ok, great :-)
I would add test covering that explicitly just to show the design intention clearly.

**Comment by electrum** [comment]
Remove backticks in commit message (they aren't needed and don't add value)

**Comment by raghavsethi** [comment]
Merged, thanks!

As an aside, I would look at the messages for the merged commits. Your messages could have been more concise while conveying the same amount of information,

**Comment by fiedukow** [comment]
@raghavsethi 
Thanks - you're right. I kinda have a bad practice of using commit message as detailed description of PR. I think it serves the purpose of PR good, but it is definitely bad for git log.
In future I will try to keep commit description compact and add detailed informations as PR description instead.
Thanks again!

**Comment by raghavsethi** [comment]
Yes, that would be ideal :)

**Pull request closed without merging**